azure-pipelines.yml

```
@@ -0,0 +1,82 @@
+name: $(TeamProject)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
```

I think it's better to build all branches
```suggestion
trigger:
  batch: true
  branches:
    include:
      - "*"

pr: none
```

---

Suggestion committed

---

Unneeded 'master' line removed